src/certificate.c

```diff
@@ -633,6 +665,11 @@ init_certificate (void)
 	Py_INCREF (&private_key_type);
 	PyModule_AddObject (module, "PrivateKey",
 			    (PyObject *) & private_key_type);
+
+	CertificateLoadingError = PyErr_NewException ("_certificate.CertificateLoadingError",
```

Comment:
It would be better to use `PyErr_NewExceptionWithDoc` and add some doc string. It would be good to add some information that it is class, etc.

Reply:
This exception will be internal now, so there is not much need for public docstring for it.

Reply:
It should have doc string despite it is only internal.